napari/layers/base/base.py

```diff
@@ -1134,3 +1134,76 @@ def _on_selection(self, selected: bool):
             self.events.select()
         else:
             self.events.deselect()
+
+    @classmethod
+    def create(
```

Comment:
`Layer.create` is a classmethod that takes the arguments in a LayerDataTuple and makes a new layer.  Seems like something we could generally use, and is useful here for the duplicate layer command.  we really ought to be able to use it to replace the `ViewerModel._add_layer_from_data` method... but the fact that `add_image` takes `channel_axis` screws up that otherwise clean "create one layer then add it" pattern.

Reply:
> but the fact that add_image takes channel_axis screws up that otherwise clean "create one layer then add it" pattern.

It is a little funny. We could deprecate and make a two line API, first add, then split and have GUI users be able to use context menu. Would you be supportive @jni? It would simplify a lot of code. It's also a pattern we've decided to use no where else in the code base.

Reply:
Ooof. This is really really widely used and also quite nice...

But yeah, `layer = split_channels(viewer.add_image(...), channel_axis=2, colormaps=[...])` is pretty nice and explicit! And honestly not that much more onerous. I'll bring it up at the eurasia meeting to get a temperature read. I think this is probably one of our most used APIs so maybe a longer deprecation like `gui_qt` is in order.

Reply:
fwiw we discussed this at the Eurasia meeting and people were supportive.

Reply:
Let's save this for a followup PR